include/myst/options.h

```diff
@@ -28,7 +22,6 @@ typedef struct myst_options
     char rootfs[PATH_MAX];
 
     myst_host_enc_uid_gid_mappings host_enc_uid_gid_mappings;
-    myst_mount_mapping_t mount_mapping;
```

Comment:
I was confused because it was not called ``mount_mappings`` (plural).

Reply:
this is deleted code!